react/ContextHeader/index.jsx

```diff
@@ -36,7 +36,7 @@ const ContextHeader = ({ icon, title, text, onClose, className }) => {
 }
 
 ContextHeader.propTypes = {
-  icon: PropTypes.node,
+  icon: iconPropType,
```

Comment:
Hmm `icon: PropTypes.oneOfType([PropTypes.node, iconPropType]),` ?

Reply:
Or we add node to the iconPropType ;)

Reply:
as `icon` prop is directly passed to `Icon` component, I think it shouldn't be a node.